internal/pkg/filesystem/fileloader/fileloader.go

```diff
@@ -146,6 +151,34 @@ func (l *loader) ReadJsonNetFileTo(def *filesystem.FileDef, target interface{})
 	return jsonNetFile, nil
 }
 
+// ReadSubDirs filter out ignored directories.
+func (l *loader) ReadSubDirs(fs filesystem.Fs, root string) ([]string, error) {
+	subDirs, err := filesystem.ReadSubDirs(fs, root)
+	if err != nil {
+		return nil, err
+	}
+	res := make([]string, 0)
+	for _, subDir := range subDirs {
+		// Ignore dirs marked with isIgnored flag
+		fileDef := filesystem.NewFileDef(filesystem.Join(root, subDir, KbcDirFileName))
+		// fileDef.AddTag(model.FileTypeJson)
```

Comment:
@webrouse když tam přidám to tagování a naimportuje se `model`, tak tam vznikne importní cyklus, máš nějaký tip jak to co nejlíp odstranit, nebo to mám nějak zkusit? 🙂
![CleanShot 2022-04-27 at 13 15 54](https://user-images.githubusercontent.com/215660/165506715-8fb674c6-187a-4818-ace7-b7351495669a.jpg)

Reply:
pozriem na to, tak za 20 min
... zatial mozes pokracovat a dat tam namiesto tej konstanty string

Reply:
Zostal teda uz iba tento problem? Alebo je este nieco ine, co mam pozriet skor?